pkg/inject/inject.go

```diff
@@ -427,16 +414,15 @@ func (conf *ResourceConfig) injectPodSpec(patch *Patch) {
 	}
 
 	if v := conf.pod.meta.Annotations[k8s.ProxyEnableDebugAnnotation]; v != "" {
-		debugEnabled, err := strconv.ParseBool(v)
+		debug, err := strconv.ParseBool(v)
 		if err != nil {
 			log.Warnf("unrecognized value used for the %s annotation: %s", k8s.ProxyEnableDebugAnnotation, v)
-			debugEnabled = false
+			debug = false
```

Comment:
I think we don't need to have this here since `strconv.ParseBool(v)` already sets debug to false if it fails.

Reply:
Also, what's the reasoning for changing the variable name from `debugEnabled` to `debug`?

Reply:
> I think we don't need to have this here since strconv.ParseBool(v) already sets debug to false if it fails.

I don't think we want to rely on that behavior.. In general, when `err` is non-nil, you shouldn't assume any semantics about the returned value.

Reply:
> Also, what's the reasoning for changing the variable name

I meant to update that in my last PR, to match the annotation.